libexec/tfenv-list-remote

```diff
@@ -73,4 +75,4 @@ remote_versions="$(curlw -sSf "${TFENV_REMOTE}/terraform/")" \
 
 #log 'debug' "Remote versions available: ${remote_versions}"; # Even in debug mode this is too verbose
 
-grep -o -E "[0-9]+\.[0-9]+\.[0-9]+(-(rc|beta|alpha|oci)[0-9]*)?" <<<"${remote_versions}" | uniq;
+grep -o -E "[0-9]+\.[0-9]+\.[0-9]+(-(rc|beta|alpha|oci)[0-9]*)?" <<<"${remote_versions}" | grep -- "$pattern" | uniq;
```

Comment:
What's the specific value in adding this capability into tfenv over just grepping your own pattern by any of a hundred means from the output of list-remote?

Reply:
Bearing in mind the ease of using grep to do this, and potential issues this could cause with internal calls to list-remote, I'm going to close this for now.